artifactory/commands/utils/configfile.go

```diff
@@ -248,11 +254,40 @@ func (configFile *ConfigFile) configMaven() error {
 		if err := configFile.setRepo(&configFile.Deployer.ReleaseRepo, "Set repository for release artifacts deployment", configFile.Deployer.ServerId, utils.LOCAL); err != nil {
 			return err
 		}
-		return configFile.setRepo(&configFile.Deployer.SnapshotRepo, "Set repository for snapshot artifacts deployment", configFile.Deployer.ServerId, utils.LOCAL)
+		if err := configFile.setRepo(&configFile.Deployer.SnapshotRepo, "Set repository for snapshot artifacts deployment", configFile.Deployer.ServerId, utils.LOCAL); err != nil {
+			return err
+		}
+		configFile.setIncludeExcludePatterns()
 	}
 	return nil
 }
 
+func (configFile *ConfigFile) setIncludeExcludePatterns() {
+	if !coreutils.AskYesNo("Would you like to filter the deployed artifacts?", false) {
+		return
+	}
+	fmt.Println("You may set multiple wildcard patterns, on after another. Set an empty pattern anytime to continue.")
```

Comment:
I suggest using the following template for prompting the user with exclude / include patterns:
```
You may set multiple wildcard patterns, to match the artifacts' names you'd like to include and/or exclude from being deployed.
Include pattern 1 (leave empty to continue): a*.jar
Include pattern 2 (leave empty to continue): b*.jar
Include pattern 3 (leave empty to continue): 
Exclude pattern 1 (leave empty to continue): a1*.jar
Exclude pattern 2 (leave empty to continue): b1*.jar
Exclude pattern 3 (leave empty to continue): 
```